packages/docs/src/routes/api/qwik-city/index.md

```diff
@@ -641,6 +641,31 @@ useNavigate: () => RouteNavigate;
 
 [Edit this section](https://github.com/BuilderIO/qwik/tree/main/packages/qwik-city/runtime/src/use-functions.ts)
 
+## valibot$
+
+```typescript
```

Comment:
Can we turn this into a real interactive example? Add `<CodeSandbox>` example: https://github.com/BuilderIO/qwik/edit/main/packages/docs/src/routes/docs/(qwik)/getting-started/index.mdx